### ISSUE TITLE
[ISSUE][DOCS]Material Types and Shaders Get Started code doesn't compile.

### ISSUE THREAD

**Issue by micronAMZN** [o3de/o3de.org#1278]
## Describe the issue

https://o3de.org/docs/atom-guide/look-dev/get-started-materialtypes-and-shaders/

The source code in this tutorial does not compile.

* The version number field in the `.materialtype` has to be incremented and move up one level.
* Using a source engine build, Asset Processor produces errors when compiling the `.shader` related to use of undeclared identifier `ViewSrg`.

## Impacted page(s)

https://o3de.org/docs/atom-guide/look-dev/get-started-materialtypes-and-shaders/

## Expected experience

Expect that when following the tutorial and copying and pasting source code from the topic, that the source code compiles.

**Comment by chanmosq** [comment]
Related another issue, https://github.com/o3de/o3de/issues/10517, created by @NiklasHenricson. It points out issues in the document and the workflow, as well as a workaround.

**Comment by NiklasHenricson** [comment]
> Related another issue, [o3de/o3de#10517](https://github.com/o3de/o3de/issues/10517), created by @NiklasHenricson. It points out issues in the document and the workflow, as well as a workaround.

Hi @chanmosq @sptramer @micronAMZN 

We finally found what all those issues were based on. Apparently the source code file (.azsl) needs an extra newline on line 1 for the Asset Processor to even consider it for parsing it. It's a bug, but the workaround for now is to add a newline there. See discussion at,

https://github.com/o3de/o3de/issues/11893

I committed changes in documentation via GitHub repo that are awaiting the review process and informed @sptramer about them. 

Sincerely,
Nik
